web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -136,8 +186,56 @@ function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
                           display = column.format(value)
                         }
 
+                        switch (column.id) {
+                          case 'relative_humidity': {
+                            if (rowIds['relative_humidity'].includes(idx)) {
+                              className = classes.minRH
+                              testId = `min-RH-td`
+                            }
+                            break
+                          }
+                          case 'temperature': {
+                            if (rowIds['min_temp'].includes(idx)) {
+                              className = classes.minTemperature
+                              testId = `min-temperature-td`
+                            } else if (rowIds['max_temp'].includes(idx)) {
+                              className = classes.maxTemperature
+                              testId = `max-temperature-td`
+                            }
+                            break
+                          }
+                          case 'precipitation':
```

Comment:
Not sure if this is easier to write as a cypress test or unit test but I think we should add a test that checks that the `className` and `testId` are set for either `precipitation`, `delta_precipitation` or `total_precipitation`.

The easiest way would be to pull out a utility function that accepts the columns and does the switch logic so it's easier to test with jest, then just have the component import and use it.

More component-y way would be to use some like enzyme to shallow render the component and check it with the library assertions: https://devhints.io/enzyme